romancal/lib/basic_utils.py

```diff
@@ -103,3 +103,45 @@ def __exit__(self, et, ev, tb):
         if self.handler and self.close:
             self.handler.close()
         # implicit return of None => don't swallow exceptions
+
+
+def ndarrays_to_recarray(arrays, names):
```

Comment:
Am I missing it, or do we only use this and its inverse in tests?  Can we get rid of this module and its tests and just rely on the library conversions?

Reply:
@bmorris3 I think we should be good. Thanks!